### ISSUE TITLE
Plugin doesn't work with Internet Explorer 11

### ISSUE THREAD

**Issue by Jarvil** [academic-moodle-cooperation/moodle-atto_bsgrid#16]
Hello,

Bootstrap grid -addon doesn't work with Internet explorer 11 & Moodle 2.7.7 & clean theme. Here are the problems:

<h1>Problem 1</h1>

First click doesn't select column text:
1. Start making new a grid, select 50% columns.
2. Click "column 1", you notice that the text isn't selected it only selects the "cell" column 1.
3. Click "column 1" second time, now it is selected

![1](https://cloud.githubusercontent.com/assets/15193418/10576756/714fabf4-766e-11e5-89a5-52c07440fdc4.png)

---

<h1>Problem 2</h1>

Second problem is that the cells change place when you type text in column 1.
1. When you have "Column 1" text selected type something f.ex. Part 1
2. Now you can see that the Column 1 is suddenly in the place where column 2 was before

![swap](https://cloud.githubusercontent.com/assets/15193418/10576762/7c4dd71a-766e-11e5-8991-66e9d9afc8d4.png)

---

<h1>Problem 3</h1>

After you type something the column 2 cells merge and dissappear (dissappear.png) Here is the code from the HTML-side: 

<p><div class="atto_bsgrid container-fluid"><div class="row-fluid">Part 1Part 2</div></div><br><p></p>

![dissapear](https://cloud.githubusercontent.com/assets/15193418/10576768/844b821e-766e-11e5-983b-631d93e73753.png)

After saving the topic and returting to it the cells are still dissappeared (dissappear2.png)

![dissappeared2](https://cloud.githubusercontent.com/assets/15193418/10576771/88b65130-766e-11e5-8002-039f01477e05.png)

**Comment by gthomas2** [comment]
Here's a jsbin which (when tested in IE11) also manifests the same issues:

http://jsbin.com/soquyowuku/1

Conclusion, IE11 is just as awful as all the previous versions :-)

Seriously though, I'll see what I can do to fix this for people who are stuck using this awful browser.

**Comment by Jarvil** [comment]
Hello,

Great, thanks for the help :)

**Comment by Jarvil** [comment]
Hello,

Any progress to get the plugin work with IE11? This problem was firstly reported when my Moodle version was 2.7 but not Its 2.9 but the problems still persists with IE11.

**Comment by Jarvil** [comment]
Hello,

Anyone there?

**Comment by ecraven** [comment]
Fortunately I have no IE11, so I have no way to test this. I'd recommend switching browsers, if that is at all feasible?

**Comment by Jarvil** [comment]
Hello Ecraven,

That is bad advice, many organizations/companies/schools doesn't have that option.

**Comment by ecraven** [comment]
@Jarvil using a standards-conformant browser is not bad advice, imho, however I recognize that's not always possible, thus "if at is at all feasible".

**Comment by gthomas2** [comment]
Hopefully I've got some time slated to look at the IE issue again soon.
If I had a dollar for every time I had to make IE work right...

**Comment by gthomas2** [comment]
I agree with @ecraven - switching from IE is the best advice you can give anyone IMO!
I understand however, that it's not always feasible.
I've spent an entire day on this bug and it's a core issue with Atto and has nothing to do with our plugin. Here is a link to the core bug I have reported, I recommend you up vote it if you want it fixing.

https://tracker.moodle.org/browse/MDL-54895
